content/c-sharp/concepts/arrays/terms/sort/sort.md

```diff
@@ -0,0 +1,70 @@
+---
+Title: '.Sort()'
+Description: 'Arranges the elements of an array in ascending or alphabetical order.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Arrays'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`.Sort()`** array method arranges the elements of an array in ascending or alphabetical order.
+
+## Syntax
+
+```pseudo
+Array.Sort(myArray);
+```
+
+`.Sort()` is a static method of the `Array` object. It takes one parameter, `myArray`, a one-dimensional zero-based array. The `.Sort()` method arranges the elements of the given array in ascending or alphabetical order.
+
+## Example
+
+The following example initializes the `letters` array with strings, then uses the `.Sort()` method to arrange the strings alphabetically. Finally, the `Console.Write()` method prints the sorted array to the console with a `foreach` loop:
```

Comment:
```suggestion
The following example initializes the `letters` array with strings, then uses the `.Sort()` method to arrange the strings alphabetically. Finally, a `foreach` loop passes each element of the sorted array to the `Console.Write()` method to be printed:
```

Reply:
Just a minor edit here, the original text suggests that the console.write method is ahead of the loop in the order of operations

Reply:
Yes, that is much more precise!